### PR TITLE
fix network configuration issues with prefix

### PR DESCRIPTION
**Pull request by SomwanshiAnjali**
Fixed below network configurations:
1. Remove statically added prefix value from srv/components/system/network/data/direct.sls
2. Update `netmask` and `gateway` for private_data network.

Signed-off-by: Anjali Somwanshi <anjali.somwanshi@seagate.com>

### CODE REVIEW

**Review by 83bhp — APPROVED** [review]
Looks good, as discussed remove the mtu parameter as of now, let's discuss it with Nitin and add it later on.

**Review by sradhanand-pati — APPROVED** [review]

**Review by mazinamdar — APPROVED** [review]
looks good to me

**Review by nitin-seagate — APPROVED** [review]
Looks good to me...
Where do we post changes for default MTU=9000 (Hardware)

File: api/python/provisioner/inputs.py [hunk 1]
```diff
@@ -686,6 +686,14 @@ class PrivateDataNetwork(ParamGroupInputBase):
         _param_group, descr="node data network mtu",
         default=1500
```
Comment (nitin-seagate) [inline api/python/provisioner/inputs.py:687]: The default value of MTU for hardware should be `9000`, else it should be `1500`

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

File: lr-cli/cortx_setup/commands/node/prepare/network.py [hunk 2]
```diff
@@ -129,6 +129,12 @@ def run(self, hostname=None, network_type=None, gateway=None, netmask=None,
 
         if network_type is not None:
 
+            server_type = function_run('grains.get', fun_args=['virtual'],
+                                targets=node_id)[f'{node_id}']
+            if not server_type:
+                raise Exception("server_type missing in grains")
+            mtu = '9000' if server_type == 'physical' else '1500'
```
Comment (nitin-seagate) [inline lr-cli/cortx_setup/commands/node/prepare/network.py:136]: mtu = '1500' if server_type == 'virtual' or network_type == 'management' else '9000'

**Review by nitin-seagate — CHANGES_REQUESTED** [review]

**Review by nitin-seagate — APPROVED** [review]
LGTM

**Pull request merged**